examples/elasticsearch-incremental/elasticsearch-incremental-blueprint.yaml

```diff
@@ -0,0 +1,538 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata: 
+  name: elasticsearch-incremental-blueprint 
+actions:
+  backup:
+    outputArtifacts:
+      s3Snap:
+        keyValue:
+          repoPath: '{{ .Phases.setupPhase.Output.repoPath }}'
+          snapshotName: '{{ .Phases.setupPhase.Output.snapshotName }}'
+          esClusterName: '{{ .Phases.setupPhase.Output.esClusterName }}'
+          esClusterNamespace: '{{ .Object.metadata.namespace }}'
+    phases:   
+    - func: KubeTask
+      name: setupPhase
```

Comment:
I agree that we do other things as well. But those things for example `kando output` is not the main think that we have that phase for. And `kando output` is standard statement that is expected to be in most of the phases.
The main thing according to me in that phase is setting up key store on the nodes. That is the reason I recommended that name so that reader would know what exactly it is, that this phase is doing. Otherwise they will have to read entire phase to figure out whats being done, considering how lengthy the phase is.

Also, it's not a problem if decide not to change that, I just think that it would make things more understandable.